SpatialGDK/Documentation/content/get-started/example-project/exampleproject-cloud-deployment.md

```diff
@@ -130,7 +130,10 @@ To launch a cloud deployment:
 1. Enter your project name (see [Set up your SpatialOS project name](#step-1-associate-your-game-with-a-cloud-project-name)). 
 1. In the **Assembly Name** field, enter the name you gave your assembly in the [previous step](#step-3-upload-your-workers).
 1. In the **Deployment Name** field, enter a name for your deployment. This labels the deployment in the [Console]({{urlRoot}}/content/glossary#console).
-1. Leave the Snapshot File field as it is. In the **Launch Config File** field, enter the path to `one_worker_test.json` (including the file name).
+1. Add the absolute path to the **Launch Config File**
+    * If you used the InstallGDK.bat script this is `/UnrealEngine/Samples/UnrealGDKExampleProject/spatial/one_worker_test.json`  
```

Comment:
For all three cases of needing to specify a file, can't we just say `/UnrealGDKExampleProject/spatial/one_worker_test.json` since the user should know where their example project is already at this point? Feels strange to specify the path before example project.

Reply:
@joshuahuburn I've updated with your suggested :) please let me know if that looks okay and you're happy to approve !